additional info on devices available to trial/sub (#1589)
saucelabs/sauce-docs: docs/mobile-apps/faq.md

For real devices on the public cloud, the limit is 6 hours for Appium tests and 1 hour for Espresso and XCUITest; the same applies to private cloud devices.
The limits for public cloud devices ensure that there are enough devices available for you, whereas the limits for private cloud devices ensure the longevity of your devices.

> #### **Is there a difference in devices I can use as a free trial and with a subscription when live testing?**

Reviewer: ```suggestion
#### **What is the difference between the devices available with a free trail and a subscription?**
```

Author: Please double check typo on trial - here it is trail

Reviewer: fixed, thanks.

> For free trial accounts, the only available devices you will see are denoted with a green corner flag that states trial and they will be available to you. If you are subscribed, you will have access to a vast pool of devices which exceeds 300 models and behind those models will be different versions of OS available to you. 

Reviewer: I assumed it is denoted by a flag that reads "Trail"?  
```suggestion
For free trial accounts, the available devices are denoted with a green Trial flag. If you are subscribed, you have access to a pool of over 300 device models with additional supported OS/version configurations. 
```

> Example: iPhone 12

Reviewer: I am not sure what this example is telling me in regards to the question.

Author: There were a list of OS versions under line 68. Something like: 
Example: iPhone 12
iPhone 12 - iOS 15
iPhone 12 - iOS 16
iPhone 12 - iOS 16.1 
and so on, the example should demonstrate that there isn't only 1 iPhone/device, but multiple running multiple versions of OS

Reviewer: @saucechaffe, I updated the example.

> For example, with a subscription, you can test iPhone 12 devices with any of these iOS configurations:
> iPhone 12 - iOS 15
> iPhone 12 - iOS 16
> iPhone 12 - iOS 16.1

Reviewer 2: ```suggestion
For example, with a subscription, you can test iPhone 12 devices with any of these iOS configurations:
* iPhone 12 - iOS 15
* iPhone 12 - iOS 16
* iPhone 12 - iOS 16.1
```
